app/static/tests/e2e/basic.etest.ts

```diff
@@ -85,9 +85,17 @@ test.describe("Basic app tests", () => {
 
         const plotSelector = ".wodin-right .wodin-content div.mt-4 .js-plotly-plot";
 
-        // Test line is plotted for 1 trace
+        // Test line is plotted for each of 3 traces
         const linesSelector = `${plotSelector} .scatterlayer .trace .lines path`;
-        expect((await page.locator(linesSelector).getAttribute("d"))!.startsWith("M0")).toBe(true);
+        expect((await page.locator(`:nth-match(${linesSelector}, 1)`).getAttribute("d"))!.startsWith("M0")).toBe(true);
```

Comment:
The example code has reverted back to the lorenz model, so we're back to 3 traces!